tcms/kiwi_auth/tests/test_backends.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+from django.contrib.auth.models import AnonymousUser
+from django.test.utils import override_settings
+
+
+@override_settings(AUTHENTICATION_BACKENDS=[
+    'django.contrib.auth.backends.ModelBackend',
+    'guardian.backends.ObjectPermissionBackend',
+    'tcms.kiwi_auth.backends.AnonymousViewBackend',
+])
+class TestAnonymousViewBackend(TestCase):
+    """Test AnonymousViewBackend.has_perm method"""
+
+    def test_has_perm(self):
+        user = AnonymousUser()
```

Comment:
`djngo.contrib.auth.models.AnonymousUser` is different than User with name `AnonymousUser` (really settings.ANONYMOUS_USER_NAME). 

The first one is the case when ObjectPermissionBackend is not enabled, the second one comes from django-guardian so these are 2 specific test scenarios with different settings.